ui/js/directives/treeherder/main.js

```diff
@@ -151,65 +137,3 @@ treeherder.directive('bugInput', function () {
         }
     };
 });
-
-treeherder.directive('numbersOnly', function () {
```

Comment:
Unused as of https://github.com/mozilla/treeherder/commit/bb396b99#diff-686a25b367215fd8175fec553976c640L26